### PR TITLE
feat: initialize app with cookiecutter

### PR DESCRIPTION
**Pull request by bseverino**
Initialize app with https://github.com/edx/edx-cookiecutters and create a `VerifiedName` model.

- [MST-788](https://openedx.atlassian.net/browse/MST-788)
- [MST-789](https://openedx.atlassian.net/browse/MST-789)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "restructuredtext.linter.disabled": true
+}
```
Comment (schenedx) [inline .vscode/settings.json:3]: #NIT newline please

File: docs/_static/theme_overrides.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* override table width restrictions */
+.wy-table-responsive table td, .wy-table-responsive table th {
```
Comment (schenedx) [inline docs/_static/theme_overrides.css:2]: Do we need this?

File: edx_name_affirmation/templates/edx_name_affirmation/base.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{% raw %}
```
Comment (schenedx) [inline edx_name_affirmation/templates/edx_name_affirmation/base.html:1]: Don't believe we want to keep any of the HTML templates around. We want to keep this library largely backend, right?

**Review by schenedx — CHANGES_REQUESTED** [review]
See my inline feedback. Largely trying to keep the repo clean.

File: edx_name_affirmation/templates/edx_name_affirmation/base.html (hunk 3 above)
Comment (bseverino) [reply edx_name_affirmation/templates/edx_name_affirmation/base.html:1]: Yeah, I'll clean those up.

**Pull request merged**